app/assets/javascripts/annotations.js

```diff
@@ -1060,14 +1186,15 @@ var submitNewPDFAnnotation = function (comment, value, problem_id, pageInd, xRat
 }
 
 /* sets up and calls $.ajax to submit an annotation */
-var submitNewAnnotation = function (comment, shared_comment, value, problem_id, lineInd, form) {
+var submitNewAnnotation = function (comment, shared_comment, global_comment, value, problem_id, lineInd, form) {
   var newAnnotation = createAnnotation();
   newAnnotation.line = parseInt(lineInd);
   newAnnotation.comment = comment;
   newAnnotation.value = value;
   newAnnotation.problem_id = problem_id;
   newAnnotation.filename = fileNameStr;
   newAnnotation.shared_comment = shared_comment;
+  newAnnotation.global_comment = global_comment;
```

Comment:
nit: this section of all these `newAnnotation.comment = comment`, etc. feels like a lot of unnecessary code; could we just do something like
```
newAnnotation = {...newAnnotation, line: parseInt(lineInd), comment, ...,global_comment}
```